examples/tokens/token_airdrop_transaction_cancel.py

```
@@ -45,7 +46,9 @@ def setup_client():
         sys.exit(1)
 
 
-def create_account(client, operator_key, initial_balance=Hbar.from_tinybars(100_000_000)):
+def create_account(
+    client, operator_key, initial_balance=Hbar.from_tinybars(100_000_000)
```

Ruff (0.14.10)

Do not perform function call `Hbar.from_tinybars` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)